spec/source_map/clip_spec.rb

```diff
@@ -1933,7 +1948,7 @@ def foo
     expect(type.to_s).to eq('Integer')
   end
 
-  xit 'identifies tuple types' do
+  xit 'dereferences tuple types with [](idx) via literals' do
```

Comment:
This is ultimately what I'd like to get working!  Static signatures for, say, 0-10 would be a little hokey, but would allow us to resolve an enormous number of tuple dereferences in the real world.